packages/server/email/MailManagerSMTP.ts

```diff
@@ -0,0 +1,48 @@
+import nodemailer from 'nodemailer'
+import sendToSentry from '../utils/sendToSentry'
+import MailManager, {MailManagerOptions} from './MailManager'
+
+export default class MailManagerSMTP extends MailManager {
+  transport = new nodemailer.createTransport({
+    host: process.env.MAIL_SMTP_HOST,
+    port: process.env.MAIL_SMTP_PORT,
+    auth: process.env.MAIL_SMTP_USER
+      ? {
+          user: process.env.MAIL_SMTP_USER,
+          pass: process.env.MAIL_SMTP_PASSWORD
+        }
+      : undefined,
+    secure: false,
+    pool: true,
+    maxConnections: 10,
+    rateLimit: 50,
+    tls:
+      process.env.MAIL_SMTP_USE_TLS === '1'
+        ? {
+            rejectUnauthorized: false,
+            ciphers:
+              process.env.MAIL_SMTP_CIPHERS ?? 'HIGH:MEDIUM:!aNULL:!eNULL:@STRENGTH:!DH:!kEDH'
```

Comment:
could you remove the fallback value here? if the env var is missing we want to know!

Reply:
Good catch, it's removed now.